runatlantis.io/guide/testing-locally.md

```diff
@@ -40,30 +58,28 @@ URL="https://{YOUR_HOSTNAME}.ngrok.io"
 ```
 
 ## Create a Webhook Secret
-GitHub and GitLab use webhook secrets so clients can verify that the webhooks came
```

Comment:
The changes in this section are out of scope of your PR.

Reply:
I think the diff GitHub is showing here is a bit misleading. This section has moved to line 60, so the line "GitHub and GitLab use webhook secrets so clients can verify that the webhooks came" is still there on line 61.

Reply:
You have changed the line breaks and modified the lines of this section which has caused the Git diffs. These changes aren't really anything to do with the Tunnelmole detail you have added, so to make PRs easier to review, it is preferable if you don't make this changes in this PR.